packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1204,8 +1204,6 @@ export class ContainerRuntime
 				}
 			},
 			(blobPath: string) => this.garbageCollector.nodeUpdated(blobPath, "Loaded"),
-			(fromPath: string, toPath: string) =>
-				this.garbageCollector.addedOutboundReference(fromPath, toPath),
```

Comment:
Do we need to bump GC Version again? We've talked about back compat from Blob Manager perspective, but what about from GC Data perspective?  Snapshots before this change will have a bunch of outbound routes from localId -> storageId, is that a problem?  It's too late in the day for me to think it through critically :P